### PR TITLE
ADR-133: adapt scene runtime to CommonJS

### PR DESCRIPTION
**Pull request by menduz**
Implementation https://github.com/decentraland/scene-runtime/pull/48

### CODE REVIEW

File: content/ADR-133-scene-runtime.md [hunk 1]
```diff
@@ -54,38 +54,41 @@ stateDiagram
     OnUpdate --> [*]
 ```
 
-### Exposed functions
+### Exposed functions and objects
+
+The runtime for the SDK7 is compatible with [CommonJS](https://en.wikipedia.org/wiki/CommonJS)'s require to load RPC modules. This is so to enable a wide variety of bundlers to create compatible Decentraland scenes.
```
Comment (marianogoldman) [inline content/ADR-133-scene-runtime.md:59]:
```suggestion
The runtime for the SDK7 is compatible with [CommonJS](https://en.wikipedia.org/wiki/CommonJS)'s `require` to load RPC modules. This is so to enable a wide variety of bundlers to create compatible Decentraland scenes.
```

File: content/ADR-133-scene-runtime.md [hunk 2]
```diff
@@ -54,38 +54,41 @@ stateDiagram
...
+class WebSocket {}
+```
 
-> TODO: define and document naming conventions about code generation for modules
+> TODO: Document fetch and WebSocket adaptations for Decentraland Scenes
 
 ### Synchronizing scene's entities with the renderer
 
-The scenes synchronize with the renderer via the `EngineAPI.SendToRenderer` RPC using the CRDT protocol defined in [ADR-117](/rpc/ADR-117). The renderer will keep a local copy of all the entities and components required for rendering. Those components are in their majority serialized using protobuf as defined in [ADR-123](/rpc/ADR-123).
+The scenes synchronize with the renderer via the `EngineApi.crdtSendToRenderer` RPC using the CRDT protocol defined in [ADR-117](/rpc/ADR-117). The renderer will keep a local copy of all the entities and components required for rendering. Those components are in their majority serialized using protobuf as defined in [ADR-123](/rpc/ADR-123).
```
Comment (marianogoldman) [inline content/ADR-133-scene-runtime.md:89]: Lin to `/rpc/ADR-117` is broken and it links to ADR homepage.

Comment (marianogoldman) [reply content/ADR-133-scene-runtime.md:89]: Same for ADR-123.

File: content/ADR-133-scene-runtime.md [hunk 3]
```diff
@@ -54,38 +54,41 @@ stateDiagram
...
 
-The `EngineAPI.SendToRenderer` response includes a list of CRDT messages to be applied in the local scene, that is used to send back information from the renderer like the position of the player.
+The `EngineApi.crdtSendToRenderer` response includes a list of CRDT messages to be applied in the local scene, that is used to send back information from the renderer like the position of the player.
```
Comment (marianogoldman) [inline content/ADR-133-scene-runtime.md:91]:
```suggestion
The `EngineApi.crdtSendToRenderer` response includes a list of CRDT messages to be applied in the local scene, that is used to send information back from the renderer like the position of the player.
```

File: content/ADR-133-scene-runtime.md [hunk 4]
```diff
@@ -95,17 +98,15 @@ sequenceDiagram
 
   S->>S: Load the code of the scene and executes it
```
Comment (marianogoldman) [inline content/ADR-133-scene-runtime.md:99]:
```suggestion
  S->>S: Load the code of the scene and execute it
```

**Review by marianogoldman — CHANGES_REQUESTED** [review]

**Review by marianogoldman — APPROVED** [review]

File: content/ADR-133-scene-runtime.md (hunk 2 above)
Comment (menduz) [reply content/ADR-133-scene-runtime.md:89]: Quemadaso, rpc tiró

**Pull request merged**